doc/neps/nep-0047-array-api-standard.rst

```diff
@@ -533,8 +627,11 @@ between many array libraries rather than only between NumPy and one other librar
 Alternatives
 ------------
 
-
-
+It was proposed to have the NumPy array API implementation as a separate
+library from NumPy. This was rejected because keeping it separate will make it
+less likely for people to review it, and including it in NumPy itself as an
+experimental submodule will make it easier for end users and library authors
+who already depend on NumPy to access the implementation.
```

Comment:
@rgommers I noticed the "alternatives" section was empty, so I added a small paragraph. Did I miss any other motivations for including the implementation in NumPy, or other alternatives?